packages/components/PersonaCoin/src/PersonaCoin.tokens.icon.ts

```diff
@@ -7,14 +7,21 @@ import { calculateEffectiveSizes } from './PersonaCoin.helpers';
 const _iconKeyProps: (keyof IPersonaCoinTokens)[] = ['iconSize', 'size', 'coinSize'];
 
 function _buildIconStyles(tokenProps: IPersonaCoinTokens /*, theme: ITheme */): ImageProps {
-  const { iconSize } = calculateEffectiveSizes(tokenProps);
+  const { iconSize, iconStrokeWidth } = calculateEffectiveSizes(tokenProps);
+  const iconSizeAdjusted = iconSize + iconStrokeWidth * 2;
+  const iconStrokeColor = tokenProps.iconStrokeColor || 'white';
```

Comment:
The `'white` should come from theme colors.

Reply:
I changed it to `theme.colors.background`